build.zig.zon

```diff
@@ -3,6 +3,7 @@
     .version = "1.2.1",
     .paths = .{""},
     .fingerprint = 0x64407a2a0b4147e5,
+    .minimum_zig_version = "0.14.0",
```

Comment:
```suggestion
    .minimum_zig_version = "0.14.1",
```

Use the same version as [example/zig-vt/build.zig.zon](https://github.com/ghostty-org/ghostty/blob/b0e85d900ec0d383d548bf3962e65ba1851c5260/example/zig-vt/build.zig.zon#L5)